repository/src/main/java/org/apache/atlas/discovery/EntityLineageService.java

```diff
@@ -590,7 +615,7 @@ private void enqueueNeighbours(AtlasVertex currentVertex, HashMap<String, Boolea
             if (!lineageListContext.evaluateTraversalFilter(currentEdge))
                 continue;
             AtlasVertex neighbourVertex;
-            if (!isConnecterVertex)
+            if (isConnectorVertex)
```

Comment:
I did not get this, why has this been inverted? Is'nt this part of the flow for data-assets in a lineage?